extensions/kafka/src/main/java/io/deephaven/kafka/ingest/JsonNodeUtil.java

```diff
@@ -476,7 +468,15 @@ public static char getChar(final JsonNode node) {
         if (isNullOrMissingField(node)) {
             return QueryConstants.NULL_CHAR;
         }
+        if (node.isNumber()) {
+            // We don't expect this to be a common case; but if the node happens to be a number, we'll assume it's meant
```

Comment:
This is a little on the weird side.

Reply:
So, consider:

```json
{
  "my_field": 97
}
```

Let's say we say we want to consume this as a char. This PR chooses to treat it as `(char)97` (`a`). The existing code treats it as `"97".charAt(0)` (`9`). We could consider being stricter and throwing an exception, but none of the other code paths are strict. I'm happy to leave the existing code in place if we prefer that behavior.